app/models/manageiq/providers/amazon/network_manager/refresh_parser.rb

```diff
@@ -350,14 +355,16 @@ def parse_load_balancer_health_check_member(member)
   end
 
   def parse_floating_ip(ip)
-    address = uid = ip.public_ip
+    cloud_network_only = ip.domain["vpc"] ? true : false
```

Comment:
would you mind doing that in another PR? Since that logic was there from the beginning and it should not be this PR changing it.